packages/components/src/text-area/src/TextArea.css

```diff
@@ -1,14 +1,13 @@
 .o-ui-text-area {
-    font-size: var(--o-ui-fs-3);
+    font-size: var(--hop-body-md-font-size);
 }
 
 .o-ui-text-area textarea {
-    --o-ui-textarea-padding: var(--o-ui-sp-3);
     resize: vertical;
     /* Fixes a bug where Firefox adds one more row than the one specified. */
     overflow-x: hidden;
-    padding: var(--o-ui-sp-2) var(--o-ui-textarea-padding);
-    min-height: var(--o-ui-sz-11);
+    padding: 0.5rem 0.75rem;
```

Comment:
There's tokens for this, 0.75 is not avail but I round up.

Reply:
ya I was just going to say this. It should be var(--hop-space-inset-squish-md)
but it's actually already set input.css so you could also remove it. I'd maybe keep it to actually keep it in this component.